ansible/configs/sap-hana/default_vars_ec2.yml

```diff
@@ -31,8 +31,9 @@ rootfs_size_hana: 200
 
 sap_extra_device: xvdf
 sap_software_device: xvdg
-sap_software_snap_name: "sap-software-v1.0"
-sap_software_snap_id: snap-12345abc123456
+sap_software_snapshot_name: "sap-software-v1.0"
+sap_software_snapshot_id: snap-12345abc123456
+sap_software_snapshot_owner_id: "337334234598"
```

Comment:
those 2 are mandatory variables, so i'd put them in the sample vars instead, unless they could have a good default